ajax/kanban.php

```diff
@@ -190,9 +190,12 @@
     }
 } else if (($_POST['action'] ?? null) === 'move_item') {
     $checkParams(['card', 'column', 'position', 'kanban']);
-    /** @var Kanban|CommonDBTM $kanban */
+    /** @var CommonDBTM $kanban */
     $kanban = getItemForItemtype($_POST['kanban']['itemtype']);
-    $can_move = $kanban->canOrderKanbanCard($_POST['kanban']['items_id']);
+    $can_move = false;
+    if (method_exists($kanban, 'canOrderKanbanCard')) {
+        $can_move = $kanban->canOrderKanbanCard($_POST['kanban']['items_id']);
+    }
```

Comment:
```suggestion
    $can_move = $kanban->canOrderKanbanCard($_POST['kanban']['items_id']);
```

Reply:
As it's a trait method, we have to check its existence to please PHPstan.

Reply:
Traits really are just awful in general...
They give the illusion of multiple inheritance, but are worse.

I'll do some local tests later this morning.